addons/web/static/src/views/fields/boolean_toggle/boolean_toggle_field.js

```diff
@@ -10,6 +10,21 @@ export class BooleanToggleField extends BooleanField {
     get isReadonly() {
         return this.props.record.isReadonly(this.props.name);
     }
+
+    async onChange(newValue) {
+        await this.props.record.update({ [this.props.name]: newValue });
+        if (this.props.record.selected && this.props.record.model.multiEdit) {
+            return;
+        }
+        const rootRecord =
+            this.props.record.model.root instanceof this.props.record.constructor &&
+            this.props.record.model.root;
+        const isInEdition = rootRecord ? rootRecord.isInEdition : this.props.record.isInEdition;
+        // We save only if we're on view mode readonly and no readonly field modifier
+        if (!isInEdition && this.props.readonly) {
+            return this.props.record.save();
+        }
```

Comment:
here we should ask dbo. I think it could be acceptable that those "editable in readonly" fields always save directly (that's what we want for status, we may want it/accept it for the other as well). It would be easier here...

Reply:
this.props.readonly -> this.props.record.isReadonly(this.props.name)